build.rs

```diff
@@ -154,6 +155,9 @@ fn make_modules(opencv_dir: &Path) -> Result<()> {
 			.map(|e| e.trim())
 			.collect::<HashSet<_>>()
 		);
+	let feature_whitelist: HashSet<String> = HashSet::from_iter(env::vars_os().filter_map(|(k, _)|
```

Comment:
I'd say it's more conventional to call `.collect()` instead of `from_iter()`. Saves you an import and keeps the more linear call chain.